### ISSUE TITLE
[stable/prometheus-operator] Alertmanager config not loaded / crashloopback

### ISSUE THREAD

**Issue by phouri** [helm/charts#13109]
Kubernetes version: v1.11.8 (GKE)
Helm/Tiller version:  v2.13.1

**Which chart**:
stable/prometheus-operator

**What happened**:
Alertmanager is crashing, can't find the configurations we loaded into the values.

Configuration:

```
global:
      smarthost: ****
      auth_username: ****
      auth_identity: ****
      auth_password: ***
      resolve_timeout: 5m
      slack_api_url: 'https://hooks.slack.com/***'
      send_resolved: true
```

The alertmanager keeps crashing with the following error:
`msg="Loading configuration file failed" file=/etc/alertmanager/config/alertmanager.yaml err="yaml: unmarshal errors:\n line 2: field auth_identity not found in type config.plain\n line 3: field auth_password not found in type config.plain\n line 4: field auth_username not found in type config.plain\n line 6: field send_resolved not found in type config.plain\n line 8: field smarthost not found in type config.plain"`

**What you expected to happen**:
Alertmanager to load

**How to reproduce it** (as minimally and precisely as possible):
Basically did nothing but installing the helm chart, but since I couldn't find any documentation and/or other errors I guess something must have gone wrong.

**Anything else we need to know**:
I checked the secret - it got created successfully with the correct data as base64. 

Appreciate any help on the matter.

**Comment by popsonebz** [comment]
facing same issue

**Comment by phouri** [comment]
Not stale 😑

**Comment by phouri** [comment]
Not stale... although the repo might be

**Comment by phouri** [comment]
Nope.

**Comment by pramodsharma62** [comment]
Has this been resolved? I am facing the same issue.
